test/lib/ufe/testGroupCmd.py

```diff
@@ -405,7 +405,45 @@ def testGroupRelative(self):
     @unittest.skipIf(os.getenv('UFE_PREVIEW_VERSION_NUM', '0000') < '3005', 'testGroupWorld is only available in UFE preview version 0.3.5 and greater')
     def testGroupWorld(self):
         '''Verify -world flag.'''
-        pass 
+        cmds.file(new=True, force=True)
+
+         # create a stage
+        (stage, proxyShapePathStr, proxyShapeItem, contextOp) = createStage();
+
+        # create a sphere generator
+        sphereGen = SphereGenerator(3, contextOp, proxyShapePathStr)
+
+        sphere1Path = sphereGen.createSphere()
+        sphere1Prim = mayaUsd.ufe.ufePathToPrim(ufe.PathString.string(sphere1Path))
+
+        sphere2Path = sphereGen.createSphere()
+        sphere2Prim = mayaUsd.ufe.ufePathToPrim(ufe.PathString.string(sphere2Path))
+
+        sphere3Path = sphereGen.createSphere()
+        sphere3Prim = mayaUsd.ufe.ufePathToPrim(ufe.PathString.string(sphere3Path))
+
+        # group Sphere1, Sphere2, and Sphere3
+        groupName = cmds.group(ufe.PathString.string(sphere1Path),
+                               ufe.PathString.string(sphere2Path),
+                               ufe.PathString.string(sphere3Path))
+
+        # verify that groupItem has 3 children
+        groupItem = ufe.GlobalSelection.get().front()
+        groupHierarchy = ufe.Hierarchy.hierarchy(groupItem)
+        self.assertEqual(len(groupHierarchy.children()), 3)
+
+        # group Sphere2 and Sphere3 with world flag enabled.
+        # world flag puts the new group under the world
+        cmds.group("{},/group1/Sphere2".format(proxyShapePathStr), 
+                   "{},/group1/Sphere3".format(proxyShapePathStr), world=True)
+
+        # verify group2 was created under the proxyshape
+        self.assertEqual([item for item in stage.Traverse()],
+            [stage.GetPrimAtPath("/group1"),
+            stage.GetPrimAtPath("/group1/Sphere1"), 
+            stage.GetPrimAtPath("/group2"),
+            stage.GetPrimAtPath("/group2/Sphere2"),
+            stage.GetPrimAtPath("/group2/Sphere3")])
```

Comment:
simple test to verify `world` flag. 

When this flag is enabled, the new group is put under the world.